doc/tech/glossary.md

```diff
@@ -131,9 +131,20 @@ Terms not defined by Kubernetes:
 
 **Software Defined Storage (SDS)** architecture consists of the storage software that is independent from the underlying storage hardware. The storage access software provides data request interfaces (APIs) and the SDS controller software provides storage access services and networking.
 
-
 <a name="1.6"></a>
-## 1.6 Other Referenced Terminology
+## 1.6 Terminology For Decoupling Applications From Infrastructure, Platform and Other Application Functionality:
+- Decoupling, Loose Coupling = Loosely coupled system is one in which each of its components has, or makes use of, little or no knowledge of the definitions of other separate components. Loose coupling is the opposite of tight coupling. ([Wikipedia:Loose Coupling](https://en.wikipedia.org/wiki/Loose_coupling))
+- Abstraction = Process of removing smaller details or attributes in the study of systems to focus attention on topics of greater importance. It can be result of decoupling. ([Wikipedia:Abstraction](https://en.wikipedia.org/wiki/Abstraction_(computer_science)))
+- Encapsulation = Restricting of direct access to some of an object's components. ([Wikipedia:Encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)))
+- Appliance deployment model = Application has tight coupling with underlying Platform even if the application is virtualized or containerized.
+- Tenant/Cloud deployment model = Tenant brings applications that are decoupled from the platform provided by Cloud operator. Tenant and Cloud operator are normally different legal entities.
```

Comment:
I don't think we need "Tenant" 
Cloud deployment model = Applications are decoupled from the platform provided by the Cloud operator.

Reply:
Done.